addons/servicegraph/promgen/promgen.go

```diff
@@ -27,16 +27,22 @@ import (
 	"github.com/prometheus/client_golang/api/prometheus/v1"
 	"github.com/prometheus/common/model"
 
+	"strings"
```

Comment:
style nit: this should be in the top block of imports, next to `strconv`.

Reply:
Done